### PR TITLE
Ms.unused code

### LINKED ISSUE #10457: [Bug] remove prompt for restoring from backup
### What happened?

restoring from backup no longer needed or used.

https://github.com/Chia-Network/chia-blockchain/search?q=No+online+backup

### Version

1.2.12 dev376

### What platform are you using?

Linux

### What ui mode are you using?

CLI

### Relevant log output

_No response_

### PR DESCRIPTION
**Pull request by mariano54**

### CODE REVIEW

**Review by hoffmang9 — DISMISSED** [review]
lgtm pending the extra import, CI, and the mypy.ini explanation.

File: mypy.ini [hunk 1]
```diff
@@ -17,7 +17,7 @@ no_implicit_reexport = True
...
...connection,tests.util.generator_tools_testing,tests.util.keyring,tests.util.key_tool,tests.util.misc,tests.util.network,tests.util.rpc,tests.util.test_full_block_utils,tests.util.test_lock_queue,tests.util.test_network_protocol_files,tests.util.test_struct_stream,tests.wallet.cat_wallet.test_cat_lifecycle,tests.wallet.cat_wallet.test_cat_wallet,tests.wallet.cat_wallet.test_offer_lifecycle,tests.wallet.cat_wallet.test_trades,tests.wallet.did_wallet.test_did,tests.wallet.did_wallet.test_did_rpc,tests.wallet.rl_wallet.test_rl_rpc,tests.wallet.rl_wallet.test_rl_wallet,tests.wallet.rpc.test_wallet_rpc,tests.wallet.simple_sync.test_simple_sync_protocol,tests.wallet.sync.test_wallet_sync,tests.wallet.test_bech32m,tests.wallet.test_chialisp,tests.wallet.test_puzzle_store,tests.wallet.test_singleton,tests.wallet.test_singleton_lifecycle,tests.wallet.test_singleton_lifecycle_fast,tests.wallet.test_taproot,tests.wallet.test_wallet,tests.wallet.test_wallet_blockchain,tests.wallet.test_wallet_interested_store,tests.wallet.test_wallet_key_val_store,tests.wallet.test_wallet_user_store,tests.wallet_tools,tests.weight_proof.test_weight_proof,tools.analyze-chain,tools.run_block,tools.test_full_sync]
```
Comment (altendky) [inline mypy.ini:20]: `,chia.wallet.util.backup_utils` was removed as the file has been deleted and thus no longer needs to be given the loose mypy constraints we applied to existing files.

**Review by altendky — COMMENTED** [review]
I'll take the liberty and push the unused import removal.

**Review by hoffmang9 — APPROVED** [review]
lgtm

**Pull request merged**